grapher/controls/Controls.scss

```diff
@@ -13,11 +13,118 @@ $zindex-ControlsFooter: 2;
     color: $controls-color;
 }
 
+// FACET STRATEGY SELECTOR
+
+// the fake dropdown menu
 .FacetStrategyDropdown {
     width: 160px;
     font-family: "Lato";
-    font-size: 10px;
+    font-size: 12px;
     font-weight: bold;
+    border: 1px solid #afb7c1;
+    border-radius: 5px;
+    padding-left: 10px;
+    padding-right: 10px;
+}
```

Comment:
Can add `cursor: pointer` to give a hint the component is interactive.

Reply:
Done 👍